### ISSUE TITLE
Data storage?

### ISSUE THREAD

**Issue by martinfleis** [ciupava/LandUseDemonstrator#11]
Hey, I've noticed that you are purposefully ignoring many data files from being uploaded to git with 
```
# ignoring some data file formats
/data/*.pb
/data/gis
/data/*.xlsx
/data/*.xls
```
To ensure we can both work on top of same data, we need a place to store all the data and access it directly from there. If the data is small enough (which it should be for our use case), would you be okay keeping the file synced by git? All the LSOA, MSOA and admin boundaries, housing price data and such? 

I see that you have these locally but cannot run your notebooks because they're not on git.

And a minor request regarding GIS files - can they not be saved as shape files but something else? GPKG?

**Comment by darribas** [comment]
Decided to go for now with a shared OneDrive folder on the Turing cloud.

**Comment by martinfleis** [comment]
Closed in #12
